config/crd/bases/approuting.kubernetes.azure.com_nginxingresscontrollers.yaml

```diff
@@ -69,8 +89,13 @@ spec:
                 properties:
                   forceSSLRedirect:
                     default: false
-                    description: ForceSSLRedirect is a flag that sets the global value
-                      of redirects to HTTPS if there is a defined DefaultSSLCertificate
+                    description: 'ForceSSLRedirect is a flag that sets the global
```

Comment:
where is the additions here coming from? This seems like comment for the code, not comment that should flow to the CRD